source/_integrations/unifi.markdown

```diff
@@ -112,17 +112,11 @@ Entities appear for each WLAN. Changing the state of WLAN will trigger a reconfi
 
 ## Sensor
 
-These sensors are disabled by default. Enabling a sensor adds a new device to Home Assistant for every connected client in your network. This can add complexity to large networks.
-
-1. To enable the sensors, on the UniFi integration page, select **Configure**. 
-2. Go to page 3/3 and enable the sensors.
-
-Currently, it is not possible to get a global bandwidth sensor for a UDM router. The integration only supports per-device sensors.
-
-
 ### Bandwidth sensor
 
-Get entities reporting receiving and transmitting bandwidth per network client.
```

Comment:
I would still like the function description of the bw sensor to stay and be the first part of the section.

Nearly there! :)

Reply:
I am obviously very confused :confused:. Can you help me get out of the hole I dug for myself here and make a suggestion?

Reply:
Sure! Not sure a disclaimer is necessary. I think it's enough with what I wrote. Or you think it should be explicitly stated that this can generate a lot of entities?

Functionality description first. 
Then how to enable it.

Get sensor entities reporting receiving and transmitting bandwidth per network client.
To enable the bandwidth sensors, on the UniFi integration page, select **Configure**, go to page 3/3 and enable the bandwidth sensors.

Reply:
Thank you for pulling me out of this :-)